### PR TITLE
Grammar and formatting changes

### PR DESCRIPTION
**Pull request by karenswry**
## Checklist

- [ ] Change is covered by automated tests
- [ ] Benchmark results are attached (if applicable)
- [ ] The [coding guidelines] are followed
- [ ] Public API has proper documentation
- [ ] Example projects and code samples are updated if needed (in case of API changes)
- [ ] Changelog is updated if needed (in case of notable or breaking changes)

[coding guidelines]: https://github.com/cossacklabs/themis/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: docs/examples/rust/README.md [hunk 1]
```diff
@@ -45,31 +37,25 @@ Note that the arguments are passed after `--`.
 
 ## keygen
 
-This tool can be used to generate key files usable by other examples.
-
-Themis supports RSA keys for some use-cases,
-but most of the features expect ECDSA keys.
+This tool can be used to generate key files usable for other examples.
+Themis supports RSA keys for some use-cases, but most features expect ECDSA keys.
```
Comment (ilammy) [inline docs/examples/rust/README.md]:
```suggestion
```

Well, _a-a-actually_ Themis supports RSA keys everywhere where ECDSA keys are accepted. We prefer and recommend EC, though. Examples that support only EC communicate this intent.

I’m not sure that it’s possible to mix EC and RSA in Secure Session and Secure Message, but they should be fine if used exclusively.

Currently, RustThemis does not accept RSA keys in Secure Session, but that’s a... deficiency in RustThemis alone. So I’d suggest to drop this line.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+Themis is an open-source, Apache 2 licensed software, maintained by [Cossack Labs](https://www.cossacklabs.com) for building reliable cryptography in our products: [Acra](https://www.github.com/cossacklabs/acra), [Hermes](https://www.cossacklabs.com/hermes), and [Toughbase](https://www.cossacklabs.com/toughbase). You can hack it in any way you want and contribute things back if you need something more than what we provide. As a development company, we focus on implementing features that are important to our products but would gladly spend some time to make Themis useful for everybody.
 
 We highly encourage you to: 
 
   - Report any bugs and request features via [GitHub Issues](https://github.com/cossacklabs/themis/issues).
-  - Report a bug and fix it with a patch via our [GitHub Pull request](https://github.com/cossacklabs/themis/pulls) (after issuing a corresponding issue and leaving a link to pull there).
-  - Participate in the theoretical development and design of Themis in the corresponding [Google Group](https://groups.google.com/d/forum/devthemis).
```
Comment (vixentael) [inline CONTRIBUTING.md]: makes sense 👍

File: docs/examples/rust/README.md (hunk 1 above)
Comment (vixentael) [reply docs/examples/rust/README.md]: > I’d suggest to drop this line.

agree

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -8,13 +8,13 @@ assignees: ''
 ---
 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+A clear and concise description of the problem, i.e.:"I'm always frustrated when [...]."
```
[the remark this answers is not in the document]
Comment (karenswry) [reply .github/ISSUE_TEMPLATE/feature_request.md]: I.e. and e.g. are both Latin abbreviations. E.g. stands for exempli gratia and means “for example.” I.e. is the abbreviation for id est and means “in other words.” Remember that E is for example (e.g.) and that I and E are the first letters of in essence, an alternative English translation of i.e.

File: src/wrappers/themis/python/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Themis provides ready-made building components, which simplifies the usage of core cryptographic security operations.
+
+[Themis]: https://github.com/cossacklabs/themis
+
+## Quickstart
+
+PyThemis requires a native Themis library.
+Please refer to the [Quickstart guide] for the installation instructions.
+
+In-depth documentation can be found on our Documentation Server][Documentation Server].
```
Comment (vixentael) [inline src/wrappers/themis/python/README.md]: @karenswry broken link 👀

Comment (vixentael) [reply src/wrappers/themis/python/README.md]:
```suggestion
In-depth documentation can be found on our [Documentation Server].     
```

**Comment by vixentael** [comment]
Merging these changes, as these are only docs changes. 

jsthemis tests start failing, dunno why, but we'll take a look

**Pull request merged**